easybuild/easyblocks/t/turbomole.py

```diff
@@ -0,0 +1,214 @@
+"""
+EasyBuild support for TURBOMOLE,
+implemented as an easyblock
+
+@author: Sven Hansen (RWTH Aachen University)
+"""
+
+import os
+import re
+import stat
+
+from easybuild.easyblocks.generic.tarball import Tarball
+from easybuild.framework.easyblock import read_file
+from easybuild.framework.easyconfig import CUSTOM
+from easybuild.tools.config import WARN
+from easybuild.tools.build_log import EasyBuildError
+from easybuild.tools.environment import setvar
+from easybuild.tools.filetools import change_dir, adjust_permissions, apply_regex_substitutions
+from easybuild.tools.run import run_cmd
```

Comment:
@sven-hansen Sorry for the radio silence on this one...

Can you look into updating this to be compatible with EasyBuild 5.x (in which we deprecated `run_cmd` in favor of `run_shell_cmd`)?